src/main/java/bhs/devilbotz/subsystems/DriveTrain.java

```diff
@@ -320,14 +327,6 @@ public DifferentialDriveWheelSpeeds getWheelSpeeds() {
    */
   @Override
   public void simulationPeriodic() {
-    /**
-     * Simulate motors and integrated sensors
-     *
-     * @see <a
-     *     href="https://github.com/crosstheroadelec/Phoenix-Examples-Languages/blob/ccbc278d944dae78c73b342003e65138934a1112/Java%20General/DifferentialDrive_Simulation/src/main/java/frc/robot/Robot.java#L144"</a>
-     * @since 1/30/2023
-     */
-
```

Comment:
Why was this comment removed?

Reply:
It was dangling, I can reintegrate the same information into the javadoc from before the declaration of the method